src/mesh/NekRSMesh.C

```diff
@@ -98,16 +98,28 @@ NekRSMesh::NekRSMesh(const InputParameters & parameters)
 
   // save the initial mesh structure in case we are applying displacements
   // (which are additive to the initial mesh structure)
-  for (int k = 0; k < _nek_internal_mesh->Nelements; ++k)
-  {
-    int offset = k * _nek_internal_mesh->Np;
-    for (int v = 0; v < _nek_internal_mesh->Np; ++v)
-    {
-      _initial_x.push_back(_nek_internal_mesh->x[offset + v]);
-      _initial_y.push_back(_nek_internal_mesh->y[offset + v]);
-      _initial_z.push_back(_nek_internal_mesh->z[offset + v]);
-    }
-  }
+
+//  if (_moving_mesh && _volume)
```

Comment:
Can you please explain why this code writing `_initial_x/y/z` was removed? If this isn't used, I'd prefer to not version comments.

Reply:
This is still in-progress, I will let you know when this is ready for review. Sorry about the commit, but hopefully a final squash will make the commit history cleaner.

Reply:
Oh gotcha, sorry for getting ahead of the curve a bit. Let me know when you'd like a re-review.